package.json

```diff
@@ -31,12 +31,12 @@
         "docs:review": "alex packages/**/*.md",
         "docs:start": "yarn workspace documentation serve --watch",
         "find": "test -f custom-elements.json",
-        "gen-react-wrapper": "rm -fr react && lerna exec --ignore \"{@spectrum-web-components/{base,bundle,custom-vars-viewer,modal,iconset,shared,opacity-checkerboard,styles,reactive-controllers,eslint-plugin},stylelint-header,@swc-react/*,documentation,example-project-rollup,example-project-webpack,swc-templates,@types/swc}\" -- cem analyze --config ../../cem-react-wrapper.config.js && node ./scripts/generate-icon-react-wrapper.js",
+        "gen-react-wrapper": "rm -fr react && yarn changeset version && yarn changeset publish && yarn cem-react-wrapper && node ./scripts/generate-icon-react-wrapper.js",
         "get-ready": "yarn build",
         "icons": "wireit",
         "icons:ui": "wireit",
         "icons:workflow": "wireit",
-        "lerna-publish": "lerna publish --message \"chore: release new versions #publish\" --force-publish",
+        "changeset-publish": "yarn changeset version && yarn changeset publish",
```

Comment:
Does this script need `yarn changeset version`? You may just want the `yarn changeset publish` command here, but I'm not certain. [Docs are here](https://github.com/changesets/changesets/blob/main/docs/command-line-options.md#version) if you wouldn't mind double-checking!

Reply:
I read the docs and foucd out that once has to call `yarn changeset version` before `yarn changeset publish`.
`yarn changeset publish` works by going into each package, checking if the version it has in its package.json is published on npm, and if it is not, running the npm publish. If you are using pnpm as a package manager, this automatically detects it and uses pnpm publish instead.
**These commands are separate to enable you to check if the release changes are accurate.**